source/features/linkify-user-location.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/dom-utils';
+import features from '../libs/features';
+
+function addLocation(baseElement: HTMLElement): void {
+	const location = select('.octicon-location', baseElement)?.nextSibling;
+	if (!location) {
+		return;
+	}
+
+	const locationName = location.textContent!.trim();
+	const googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(locationName)}`;
+
+	wrap(location, <a href={googleMapsLink} className="text-gray" />);
```

Comment:
```suggestion
	wrap(location, <a href={googleMapsLink} />);
```

On second thought, we can leave this blue so it's clearly clickable.